src/module_wrap.cc

```diff
@@ -495,20 +495,15 @@ Maybe<uv_file> CheckFile(const std::string& path,
 
 const PackageConfig& GetPackageConfig(Environment* env,
                                                    const std::string path) {
-  static const PackageConfig kEmptyPackage =
-      { false, true, false, "" };
-  static const PackageConfig kInvalidPackage =
-      { true, false, false, "" };
-
   auto existing = env->package_json_cache.find(path);
   if (existing != env->package_json_cache.end()) {
     return existing->second;
   }
   Maybe<uv_file> check = CheckFile(path, LEAVE_OPEN_AFTER_CHECK);
   if (check.IsNothing()) {
-    env->package_json_cache.emplace(path,
+    auto entry = env->package_json_cache.emplace(path,
         PackageConfig{ false, true, false, "" });
-    return kEmptyPackage;
+    return (*entry.first).second;
```

Comment:
Any reason this isn't using `->`?

Reply:
That would be the sensible way to write this yes :)